corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
     visibility:
       type: string
       enum: [public,private]
+    project:
+      type: object
+        properties:
+          description:
+            type: string
+          name:
+            type: string
+          id:
+            $ref: '#/components/schemas/project_uuid'
+          s3_bucket_key:
+            type: string
+          submitter:
+            type: object
+            properties:
+              id:
+                $ref: '#/components/schemas/user_uuid'
+              email:
+                type: string
+          links:
+            type: object
+            properties:
+              protocol:
+                type: array
+                items:
+                  type: string
+              other:
+                type: array
+                items:
+                  type: string
+              raw_data:
+                type: array
+                items:
+                  type: string
+              attestation:
```

Comment:
attestation is a different thing than links - it's directly related to a project, referencing a form that needs to be filled out. There is also only one per project so an array doesn't make sense here.

Reply:
so it's not a link to a form?

Reply:
It's an S3 URI so yes technically a link? I see that you've separated out attestation though so I'm uncertain whether the comment is still applicable?